### ISSUE TITLE
parameter setting

### ISSUE THREAD

**Issue by LittleJJIU** [jbr97/URM#2]
Hello, I have a question regarding the parameter settings used during training. I noticed that the parameters mentioned in the paper are not consistent with the default parameters provided in the code.
And I can't find the instances_train_cate.json. Is count_train_cate.json the same as this file?
I conducted a few-shot A → B experiment. Here are the results I obtained using the default code settings:

Epoch: 121
Train loss: 0.000
Aux train loss: 0.001
Train distill language loss: 0.258
Val loss: 0.031
Train MAE: 16.846
Val MAE: 37.320
Best MAE: 23.970
Best RMSE: 36.807
Epoch time: 326.250 seconds

And here are the results using the --lr and --weight_decay parameters specified in the paper:

Epoch: 151
Train loss: 0.000
Aux train loss: 0.000
Train distill language loss: 0.513
Val loss: 0.061
Train MAE: 7.575
Val MAE: 37.852
Best MAE: 23.126
Best RMSE: 35.561
Epoch time: 479.622 seconds

The difference in results seems relatively small.
